### DISCUSSION TITLE
Multi-class registration for 2D registration

### DISCUSSION THREAD

**Discussion by ku294714** [Project-MONAI/MONAI#2633] [Q&A]
Hello all,

I would like to use 2 class labels for 2D registration task.

Do I need to add a channel in the label tensor to make it as (batchsize  *  2 *  dim1  *  dim2) shape?

Also, what is the difference between globalnet (used in 2D registration) and localnet (used in 3D registration)?

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Could you please help share some comments here? I think you are expert on registration tutorials.

Thanks in advance.

**Reply by ku294714** [reply]
Hello Dear @Nic-Ma ,

Please let me know about the solution of this issue.

**Reply by Nic-Ma** [reply]
Hi @wyli , I am not very sure about the `GlobalNet` and `LocalNet`, could you please help share some comments here?

Thanks in advance.

**Reply by wyli** [reply]
for the network definitions please see the documentation
- https://docs.monai.io/en/latest/networks.html#monai.networks.nets.GlobalNet
- https://docs.monai.io/en/latest/networks.html#monai.networks.nets.LocalNet

**Reply by ku294714** [reply]
Thank you for the reply @wyli . Can you ask tell me about multi class registration? For 2D registration, I want to use 2 labels. Do I need to add a channel in the label tensor to make it as (batchsize * 2 * dim1 * dim2) shape?

**Reply by wyli** [reply]
Yes I guess the label tensor should be in that shape so that it could be warped

**Reply by ku294714** [reply]
Thank you.
